helpers/mongodb

```diff
@@ -0,0 +1,360 @@
+#!/bin/bash
+
+readonly YNH_DEFAULT_MONGO_VERSION=5.0
+# Declare the actual MongoDB version to use: 4.4 ; 5.0 ; 6.0
+# A packager willing to use another version of MongoDB can override the variable into its _common.sh.
+YNH_MONGO_VERSION=${YNH_MONGO_VERSION:-$YNH_DEFAULT_MONGO_VERSION}
```

Comment:
This is going to cause stupid boring issues because of apps in the wild using the experimental helper, and actually people manually tweaked the "YNH_DEFAULT_MONGO_VERSION" instead of defining "YNH_MONGO_VERSION" and it's a total mess. Also it will crash because calling `readonly foo=bar` two times (one in the official helper, one in the custom experimental helper existing in the wild) is not valid

But the good news is that all this is just a shortcut to have a default value for the `--mongo_version` arg of `ynh_install_mongo`

So let's just git of this and assume packagers should define a `$mongo_version` var and call `ynh_install_mongo --mongo_version=$mongo_version`, just like nodejs helper etc. Some mongodb apps already do this.

```suggestion
```